cpp/src/parquet/properties.h

```diff
@@ -1256,12 +1257,28 @@ class PARQUET_EXPORT ArrowWriterProperties {
       return this;
     }
 
+    /// \brief Write time32 and time64 columns whose isAdjustedToUTC field is true.
+    ///
+    /// \note Does not apply to timestamp columns.
+    Builder* enable_time_adjusted_to_utc() {
+      write_time_adjusted_to_utc_ = true;
+      return this;
+    }
```

Comment:
How about exposing a single setter instead?
```suggestion
    Builder* set_time_adjusted_to_utc(bool adjusted) {
      time_adjusted_to_utc_ = adjusted;
      return this;
    }
```

Reply:
Will do!

I was following the existing  `enable_*` and `disable_*` pattern used for other  `true|false` properties, but adding a method that accepts a `bool` input argument is more efficient.